### PR TITLE
logkit tailx采集rotate日志丢数据

### PR DESCRIPTION
**Pull request by shangmin-001**
## Changes
   
- [ ] feature1
- [ ] feature2
- [ ] fixbug1
- [ ] fixbug2
 
## Reviewers

- [ ] @[someone] please review
- [ ] @[someotherone] please review

## Wiki Changes
 
- options1...
- options2...
    
## Checklist
   
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] Wiki updated

### CODE REVIEW

File: reader/singlefile/singlefile.go [hunk 1]
```diff
@@ -274,6 +274,16 @@ func (sf *SingleFile) Reopen() (err error) {
 	if newInode == oldInode {
 		return
 	}
+	endOffset, err := sf.f.Seek(0, 2)
```
Comment (PapaPiya) [inline reader/singlefile/singlefile.go:277]: 用io.SeekStart/io.SeekEnd更清楚一点

**Comment by PapaPiya** [comment]
go format 下

**Comment by shangmin-001** [comment]
> go format 下

ok

**Review by PapaPiya — APPROVED** [review]
/lgtm

**Pull request merged**